### ISSUE TITLE
Lab 2 - Exercise 0 - Select Workspace

### ISSUE THREAD

**Issue by VitalyMCT** [MicrosoftLearning/DP-203T00-Data-Engineering-on-Microsoft-Azure#84]
A workspace needs to be selected prior to being able to proceed to the Manage hub:

![image](https://user-images.githubusercontent.com/1765732/118728191-9ed53280-b7e8-11eb-9d89-cc149ad2d039.png)

**Comment by ctesta-oneillmsft** [comment]
unable to reproduce in the environment

**Comment by ctesta-oneillmsft** [comment]
Correct. the ALH environments provided to support this course already do this step as part of the setup, so the students wont see this. You will only see this if you deploy the course in your own environment

**Comment by VitalyMCT** [comment]
This was in an ALH (CloudLabs). Are you saying their setup script may have malfunctioned?

**Comment by ctesta-oneillmsft** [comment]
OK . I will follow up with them thanks

**Comment by VitalyMCT** [comment]
Running a live class via CloudLabs now. This is still an issue for the students.
